resources/definitions/Ender3_MicroSwiss_DirectDrive.def.json

```
@@ -0,0 +1,95 @@
+{
+    "name": "Ender3 MicroSwiss DirectDrive",
+    "version": 2,
+    "inherits": "creality_base",
+    "metadata": {
+        "quality_definition": "creality_base",
+        "visible": true,
+        "platform": "creality_ender3.3mf",
+        "author": "GitHub.com/RFBomb",
+
+        "machine_extruder_trains": {
+            "0": "MicroSwiss_DirectDrive_extruder_0",
+            "1": "MicroSwiss_DirectDrive_extruder_1"
+        }
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "Ender-3 MicroSwiss DirectDrive" },
+        "machine_extruder_count": { "value": 1, "maximum_value": "2" },
+        "extruders_enabled_count": { "value": 1 },
+        "machine_use_extruder_offset_to_offset_coords": { "value": true },
+        
+        "machine_width": { "value": 230 },
+        "machine_depth": { "value": 230 },
+        "machine_height": { "value": 175 },
+        "gantry_height": { "value": 35 },
+
+        "machine_disallowed_areas": {
+            "description": "A list of polygons with areas the print head is not allowed to enter.",
+            "default_value": [
+                  [[-117.5, 117.5], [-117.5, 108], [117.5, 108], [117.5, 117.5]],
+                  [[-117.5, -108], [-117.5, -117.5], [117.5, -117.5], [117.5, -108]]
+             ]
+        },
+
+        "machine_head_with_fans_polygon": { "default_value": [
+                [-26, 34],
+                [-26, -32],
+                [32, -32],
+                [32, 34]
+            ]
+        },
+        
+        
+        
+        "machine_firmware_retract":  { "default_value": true },
```

This should override the `value` and not the `default_value` (according to the failing tests).
```suggestion
        "machine_firmware_retract":  { "value": true },
```